qa/suites/rados/thrash/d-require-luminous/at-end.yaml

```diff
@@ -21,5 +21,11 @@ overrides:
     conf:
       global:
         mon debug no require luminous: true
+
+# setting luminous triggers peering, which *might* trigger health alerts
+    log-whitelist:
+      - overall HEALTH_
+      - (PG_AVAILABILITY)
```

Comment:
hmm, all of the whitelists are like this.  i'll fix them all at once in a follow-up pr